meetup/services/meetup_presenter.py

```diff
@@ -0,0 +1,42 @@
+import time
+from pizzaplace.services.pizza_place_presenter import PizzaPlacePresenter
```

Comment:
"Services" doesn't seem like quite the right module for this. What about "presenters"?